syntax = "proto3";

package fixbug;

// 定义下面选项，表明生成service服务类和rpc方法，默认不生成
option cc_generic_services = true;

message Resultcode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

message GetFriendsListRequest
{
    uint32 userid = 1;
}


message GetFriendsListResponse
{
    Resultcode result = 1;
    repeated bytes friends = 2;
}

// 好友模块
service FriendServiceRpc
{
    rpc  GetFriendsLists(GetFriendsListRequest) returns(GetFriendsListResponse);
}